lib/form/paragraph3.flow

```diff
@@ -197,20 +230,31 @@ renderParagraph(
 		})
 	));
 
+	size = make(WidthHeight(maxW, 0.0));
 	Constructor(
-		Inspect(
-			[AvailableWidth2(aw)],
-			if (paragraphBorder.top > 0.0 || paragraphBorder.bottom > 0.0) {
-				Border(0.0, paragraphBorder.top, 0.0, paragraphBorder.bottom, contentGroup);
-			} else {
-				contentGroup;
-			}
-		),
+		{
+			inspector = Inspect(
+				[AvailableWidth(aw)], 
+				if (paragraphBorder.top > 0.0 || paragraphBorder.bottom > 0.0) {
+					Border(0.0, paragraphBorder.top, 0.0, paragraphBorder.bottom, contentGroup);
+				} else {
+					contentGroup;
+				}
+			)
+			// Available2, Size2 — neither works, extra space allocated,
+			// invalid behaviour, visible only when other fillers present in TCols.
+			/*Select(aw, \awv -> {
```

Comment:
what is it aboud?

Reply:
This was about cropping does not solve the issue, I wanted you to see this, and, maybe, commit until next PR ready which solves this issue.